README.md

```diff
@@ -69,6 +69,10 @@ Learn by example:
 
 ; Custom registered keywords always takes precedence over inference
 (sc/coerce ::my-custom-attr "Z") ; => #user.SomeClass{:x "Z"}
+
+; Coercers in the registry can be overriden within a specific context
+(binding [sc/*overrides* {::my-custom-attr keyword}]
```

Comment:
```suggestion
(binding [sc/*overrides* {::my-custom-attr keyword?}]
```

Reply:
Actually no.

The value of the overridden keys must be a coercing function, not a predicate. In this case `keyword`. One can use the inference of the library if desired ``(sc/sym->coercer `keyword?)`` to find the coercion function for a predicate, but any function that does the coercion is allowed.

Reply:
makes sense, thanks for clarifying!

Reply:
My pleasure.